src/components/Collections/Collection.tsx

```diff
@@ -111,59 +116,80 @@ export function Collection({
     );
   }
 
+  const collectionType = collection?.type;
+  // TODO.collections: This is tied to images for now but
+  // we will need to add a check for other resources later
+  const canAddContent =
+    collectionType === CollectionType.Image && permissions?.write && permissions?.isContributor;
```

Comment:
need to check for `write` and `writeReview`. No need for `isContributor`, as write/write_review return false when the user is not the owner or a contributor